azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatAdapter.java

```diff
@@ -4,42 +4,50 @@
 package com.azure.android.communication.ui.chat;
 
 import android.content.Context;
+import android.content.Intent;
 
 import com.azure.android.communication.common.CommunicationIdentifier;
 import com.azure.android.communication.common.CommunicationTokenCredential;
 import com.azure.android.communication.ui.chat.configuration.ChatCompositeConfiguration;
 import com.azure.android.communication.ui.chat.models.ChatCompositeErrorEvent;
+import com.azure.android.communication.ui.chat.models.ChatCompositeRemoteOptions;
+import com.azure.android.communication.ui.chat.presentation.ChatCompositeActivityImpl;
+
+import java9.util.concurrent.CompletableFuture;
 
 /**
  * Azure android communication chat composite component.
  *
  * <p><strong>Instantiating Chat Composite</strong></p>
  */
-public final class ChatUIClient {
+public final class ChatAdapter {
 
     private static int instanceIdCounter = 0;
     final Integer instanceId = instanceIdCounter++;
-    private final Context applicationContext;
+    private ChatContainer chatContainer;
     private final String endpoint;
     private final CommunicationIdentifier identity;
     private final CommunicationTokenCredential credential;
+    private final String threadId;
     private final String displayName;
     private final ChatCompositeConfiguration configuration;
 
-    ChatUIClient(final Context applicationContext,
-            final ChatCompositeConfiguration configuration,
-                 final String endpoint,
-                 final CommunicationIdentifier identity,
-                 final CommunicationTokenCredential credential,
-                 final String displayName) {
-        this.applicationContext = applicationContext;
+    ChatAdapter(final ChatCompositeConfiguration configuration,
+                final String endpoint,
+                final CommunicationIdentifier identity,
+                final CommunicationTokenCredential credential,
+                final String threadId,
+                final String displayName) {
```

Comment:
The order of the arguments is a little different but it should be fine I can change it on the iOS side